elements/pl-string-input/pl-string-input.py

```diff
@@ -243,8 +243,8 @@ def grade(element_html, data):
 
     # Remove the blank spaces between characters
     if (remove_spaces):
-        a_sub = a_sub.replace(' ', '')
-        a_tru = a_tru.replace(' ', '')
+        a_sub = ''.join(a_sub.split())
+        a_tru = ''.join(a_tru.split())
```

Comment:
This is consistent with the usage of `strip()` above for `remove-leading-trailing`, but in the future we might consider using a regex with `\s` instead, which will handler a wider variety of whitespace characters. Not requesting a change here though!